index.bs

```diff
@@ -179,13 +179,27 @@ a number of properties for itself:
 *   <dfn for="user agent" export>full version</dfn> (for example: "72.0.3245.12", "3.14159", or "297.70E04154A")
 *   <dfn for="user agent" export>platform brand</dfn> (for example: "Windows", "iOS", or "AmazingOS")
 *   <dfn for="user agent" export>platform version</dfn> (for example: "NT 6.0", "15", or "17G")
-*   <dfn for="user agent" export>platform architecture</dfn> (for example: "ARM64", or "ia32")
+*   <dfn for="user agent" export>platform architecture</dfn> (for example: "arm", or "x86")
 *   <dfn for="user agent" export>model</dfn> (for example: "", or "Pixel 2 XL")
 *   <dfn for="user agent" export>mobileness</dfn> (for example: ?0 or ?1)
 
 [=User agents=] SHOULD keep these strings short and to the point, but servers MUST accept arbitrary
 values for each, as they are all values constructed at the [=user agent=]'s whim.
 
+[=User agents=] MUST map higher-entropy [=platform architecture=] values to the following buckets:
+
+* 32 bit x86 CPU architectures => "x86"
+* 64 bit x86 CPU architectures => "x86_64"
+* 32 bit ARM CPU architectures => "arm"
+* 64 bit ARM CPU architectures => "arm_64"
```

Comment:
This architecture is conventionally called arm64, not arm_64. (Unlike the 64-bit version of x86, which is known by many names, where x86_64 is one of the common ones.

Reply:
Yeah, I guess we could stick with current conventions. Changed it.